### ISSUE TITLE
Apply TD3_plus_demos on the tasks like TwoEggCatchUnderArm-v0

### ISSUE THREAD

**Issue by Jianshu-Hu** [henrycharlesworth/SCDM#2]
I was trying to apply the method of TD3_plus_demos on the tasks like TwoEggCatchUnderArm-v0 and EggCatchUnderarm-v0. I copied the demostrations from the folder `/prerun_trajectories` to `/TD3_plus_demos`, modified corresponding directories and run the `main.py`. I noticed that there is not any improvement after training for long time steps. I would appreciate it if you can give some advice. Thanks in advance!

**Comment by Jianshu-Hu** [comment]
These were the results I got.
![Training results](https://user-images.githubusercontent.com/58180987/136640463-670e92d9-1f13-4ff3-914e-5ff3703174e5.png)

**Comment by henrycharlesworth** [comment]
Hi, we never ran any experiments with TD3 plus demos on any of the goal based environments (we only used it for penspin). I'm not necessarily that surprised it doesn't work because I don't think TD3 with dense rewards works well on any goal-based dexterous manipulation task. It would be interesting to try combining HER with demos in a similar way which might work a bit better, I did run some initial experiments along these lines but never fully investigated it.
